snapshotter/sys.config

```diff
@@ -47,9 +47,9 @@
    {fetch_latest_from_snap_source, true},
    {honor_quick_sync, true},
    {quick_sync_mode, blessed_snapshot},
-   {blessed_snapshot_block_height, 1160641},
+   {blessed_snapshot_block_height, 1318321},
    {blessed_snapshot_block_hash,
-    <<114,177,177,98,217,154,188,18,212,229,36,77,57,49,35,131,227,60,157,115,47,34,29,59,241,63,24,152,221,198,197,5>>},
+    <<34,205,162,107,155,68,156,241,133,93,191,151,136,126,186,148,108,195,13,62,106,205,175,236,235,131,189,106,249,216,149,117>>},
```

Comment:
Block height and hash are taken from upstream sys.config. They come out different if I use generate script. I am not sure about the relationship between this and generated file.

Reply:
Agreed. I also believe these values are ultimately overwritten, but hard to follow every possible code path. Might as well update.